webpack/assets/javascripts/react_app/routes/common/reducerHOC/withQueryReducer.js

```diff
@@ -2,6 +2,7 @@ import Immutable from 'seamless-immutable';
 
 const initialState = Immutable({
   page: 1,
+  perPage: 20,
```

Comment:
This should be the value defined by the setting, not hard coded.

Reply:
It will get updated after a moment 
before the table is mounted by the `AUDIT_PAGE_UPDATE_QUERY` action
and anyway the pagination default per page is 20.
but this will make sure the `perPage` isn't `undefined`.

cc @glekner what are your thoughts?

Reply:
In the [Query PR](https://github.com/theforeman/foreman/pull/6839/files#r292507052) We decided not to have `perPage` in the initial state. But it seems like this error wasn't present back then.

It's true that `AUDIT_PAGE_UPDATE_QUERY` will update to the correct `perPage` anyway, so `20` is fine by me.

Reply:
until `AUDIT_PAGE_UPDATE_QUERY` is finished the page is loading, so how about giving it an undefined as an initial value? it doesn't make sense if at the first rendering it will be a falsy 20, and after an API call, it will be updated to the correct value.